packages/one-app-bundler/bin/bundle-module.js

```diff
@@ -13,22 +13,32 @@
  * under the License.
  */
 
-if (process.env.NODE_ENV !== 'production' && process.argv.includes('--dev')) {
-  console.info('Running dev bundler');
-  import('@americanexpress/one-app-dev-bundler').then(({ default: esbuildBundleModule }) => {
-    if (esbuildBundleModule) {
-      esbuildBundleModule().catch((error) => {
+const bundleModule = async () => {
+  const {
+    devBuildModule,
+    bundleExternalFallbacks,
+  } = await import('@americanexpress/one-app-dev-bundler');
+
+  await bundleExternalFallbacks();
```

Comment:
By performing this step here is it true that
* If the user edits their package.json externals config, they need to restart the bundler
* If the user installs a newer version of a package, they need to restart the bundler

Are there any other thing about not including this in the normal `watch` pipeline?

(The two above stipulations are OK IMHO, since i dont think `watch` watches the package.json or node_modules anyway, just checking that I understand it fully)

Reply:
yes that is correct.